helmfile.d/cert-manager.yaml

```diff
@@ -1,11 +1,17 @@
 releases:
+    - name: cert-manager
+      chart: jetstack/cert-manager
+      namespace: cert-manager
+      wait: true
+      version: v0.15.1
+      set:
+        - name: installCRDs
+          value: true
+
     - name: acme
       chart: ../charts/acme
```

Comment:
I thought this one inherits from cert-manager. Should it be both of them in the helmfile?

Reply:
The idea is to manage crd as part of the helm release, [doc](https://cert-manager.io/docs/installation/kubernetes/#steps)